### PR TITLE
examples: comment improvement in remote_cluster.py and deployment_cre…

### PR DESCRIPTION
**Pull request by jonasdlindner**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Provide an uniform comment format for all examples

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: examples/remote_cluster.py [hunk 1]
```diff
@@ -12,9 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This example demonstrate communication with a remote Kube cluster from a
-# server outside of the cluster without kube client installed on it.
-# The communication is secured with the use of Bearer token.
+"""
+This example demonstrates the communication between a remote cluster and a
+server outside the cluster without kube-client installed on it.
```
Comment (yliaog) [inline examples/remote_cluster.py:17]: s/kube-client/kube client/

Comment (jonasdlindner) [reply examples/remote_cluster.py]: adjusted to kube client

**Comment by yliaog** [comment]
please squash the commits, lgtm otherwise.

**Pull request merged**